apps/institution-service/cypress/e2e/institutions.cy.ts

```diff
@@ -131,15 +202,33 @@ describe("POST /institutions (Institution create)", () => {
     url: `http://localhost:${PORT}/institutions`,
     method: "POST",
   });
+
   runInvalidPermissionCheck({
     url: `http://localhost:${PORT}/institutions`,
     token_env_var: "USER_ACCESS_TOKEN",
     method: "POST",
+    body: testInstitution,
   });
 
-  it("gets 201 created when user has permission to create", () => {
-    const getUniqueId = () => Cypress._.uniqueId(Date.now().toString());
+  createValidationTestCases.forEach((testCase) => {
+    it(`should fail validation when ${testCase.description}`, () => {
+      cy.request({
+        url: `http://localhost:${PORT}/institutions`,
+        method: "POST",
+        body: testCase.body,
```

Comment:
It feels like it would be less complex to generate the ID here instead of running through array in the reduce above